docs/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+<!--
+Thank you for your pull request!
+
+Please see https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md for guidelines on
+how to best contribute to the Windows App SDK Samples!
+
+-->
+
+##### Checklist
+
+- [ ] Required: Please ensure youre samples can be correctly built using Visual Studio 2019 and above
```

Comment:
>Visual Studio 2019 and above [](http://example.com/codeflow?start=73&length=28)

maybe this instead? that way we don't have to come back here and update this later when we change the min version?
"the Visual Studio versions in [https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=stable#2-install-visual-studio](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=stable#2-install-visual-studio)"